dask/dataframe/dask_expr/_util.py

```diff
@@ -226,3 +233,11 @@ def _is_any_real_numeric_dtype(arr_or_dtype):
 def get_specified_shuffle(shuffle_method):
     # Take the config shuffle if given, otherwise defer evaluation until optimize
     return shuffle_method or config.get("dataframe.shuffle.method", None)
+
+
+if azure_identity_installed:
+    from azure.identity import DefaultAzureCredential
+
+    @normalize_token.register(DefaultAzureCredential)
+    def tokenize_azure_credential(obj):
```

Comment:
We typically use `register_lazy`

https://github.com/dask/dask/blob/2df9ed342d7dc8fc774c6b493310aaae8036711c/dask/utils.py#L730